neo/Consensus/ConsensusService.cs

```diff
@@ -263,14 +263,10 @@ private void OnConsensusPayload(ConsensusPayload payload)
                 return;
             }
             if (payload.ValidatorIndex >= context.Validators.Length) return;
-            ConsensusMessage message = payload.ConsensusMessage;
-            if (message.ViewNumber != context.ViewNumber && (message.Type == ConsensusMessageType.PrepareRequest ||
```

Comment:
Oh, you moved these into the methods instead. I just added it back thinking it was a merge issue. I'll remove it since you added it in the methods.

Reply:
Ok, reverted my change. This is resolved.